ui/page/components/wallet_setup_page.go

```diff
@@ -227,9 +234,17 @@ func (pg *CreateWallet) Layout(gtx C) D {
 								Top:   values.MarginPadding26,
 								Right: values.MarginPadding20,
 							}.Layout(gtx, func(gtx C) D {
-								return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
-									layout.Rigid(pg.walletTypeSection),
-									layout.Rigid(pg.walletOptions),
+								return layout.Stack{}.Layout(gtx,
+									layout.Expanded(func(gtx layout.Context) layout.Dimensions {
+										return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+											layout.Rigid(pg.walletOptions),
+										)
+									}),
+									layout.Expanded(func(gtx layout.Context) layout.Dimensions {
+										return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+											layout.Rigid(pg.walletTypeSection),
+										)
+									}),
```

Comment:
I don't see any reason use `layout.Flex` in here

```suggestion
									layout.Expanded(pg.walletOptions),
									layout.Expanded(pg.walletTypeSection),
```